linux_new/working_v3.groovy

```diff
@@ -0,0 +1,795 @@
+                            } else {
+                                 error("The version string format does not match the expected pattern.")
+                        }
+                    } else {
+                        echo "Not JDK8"
+                    // Parse The Version Tag, Into Usable Components
+                    
+                    def split = (params.Tag =~ /(jdk)-(\d[\d.]*)([+_]\d+)?/)
+
+                    if (split.find()) {
+                        vername = split.group(1)
+                        verversion = split.group(2)
+                        verbuild = split.group(3)?.replaceAll("[+_]", "")
+                    } else {
+                        error("The version string format does not match the expected pattern.")
+                    }
+
+                    }
+
+                    // echo "Debug 01"
+                    // echo "01 - Vername = ${vername}"
+                    // echo "02 - VerVer  = ${verversion}"
+                    // echo "03 - VerBLD  = ${verbuild}"
+
+                    // Construct the Filename
+                    def filennameFinal =""
+                    def filenamePrefix = "Open"
+                    def filenameSuffix = "tar.gz"
+                    def filenameVersion = params.Version.toUpperCase() + "U"
+                    def packageTypes = ['jdk', 'jre']
+
+                    packageTypes.each { packageType ->
+                      if (params.Tag?.startsWith("jdk8")) {
+                        filenameFinal = "${filenamePrefix}${filenameVersion}-${packageType}_${arch}_${distro}_${JVM}_${verversion}${verbuild ?: 'N/A'}"
+                        ReleaseVersion = "${verversion}${verbuild ?: 'N/A'}"
+                      } else {
+                        filenameFinal = "${filenamePrefix}${filenameVersion}-${packageType}_${arch}_${distro}_${JVM}_${verversion}_${verbuild ?: 'N/A'}"
+                        ReleaseVersion = "${verversion}_${verbuild ?: 'N/A'}"
+                      }
+                      
+                      def JDKFinal = "${filenameFinal}.${filenameSuffix}"
+                      def SHAFinal = "${JDKFinal}.sha256.txt"
+                      
+                      // ReleaseVersion = "${verversion}_${verbuild ?: 'N/A'}"
+                      
+                      echo "JDK File Name : ${JDKFinal}"
+                      echo "SHA File Name : ${SHAFinal}"
+                      DLfilenames << JDKFinal
+                      DLfilenames << SHAFinal
+
+                      JDKArray << [ "${packageType}" , "${JDKFinal}", "${SHAFinal}" , "${distro}" , "${arch}" , "${ReleaseVersion}" ]
+                    }
+
+                    // Set Package Release Version If Repackage
+                    if (params.RePackage) {
+                        PackageReleaseVersion = params.Package_Increment
+                      }
+
+                      echo "Debug 01 - Parameters Including Pacakage Release Version"
```

Comment:
Was this left in deliberately?

Reply:
Also, several other debug messages in this file.

Reply:
Removed the rogue debug file, checked in by accident.